cmd/psqldef/tests.yml

```diff
@@ -297,6 +297,27 @@ IndexAndForeignOrder:
     );
     CREATE UNIQUE INDEX uniq_idx ON users (id);
     ALTER TABLE posts ADD CONSTRAINT "user_id_key" FOREIGN KEY ("user_id") REFERENCES users ("id");
+ForeignKeyConstraintsAreEmittedLast:
+  current: |
+    CREATE TABLE foos ( dummy_column text );
+    CREATE TABLE bars ( dummy_column text );
+  desired: |
+    CREATE TABLE foos (
+      foo_id bigint PRIMARY KEY,
+      bar_id bigint NOT NULL REFERENCES bars (bar_id)
+    );
+    CREATE TABLE bars (
+      bar_id bigint PRIMARY KEY
+    );
+  output: |
+    ALTER TABLE "public"."foos" ADD COLUMN "foo_id" bigint NOT NULL;
+    ALTER TABLE "public"."foos" ADD COLUMN "bar_id" bigint NOT NULL;
+    ALTER TABLE "public"."foos" ADD PRIMARY KEY ("foo_id");
+    ALTER TABLE "public"."bars" ADD COLUMN "bar_id" bigint NOT NULL;
+    ALTER TABLE "public"."bars" ADD PRIMARY KEY ("bar_id");
+    ALTER TABLE "public"."foos" ADD CONSTRAINT "foos_bar_id_fkey" FOREIGN KEY ("bar_id") REFERENCES "public"."bars" ("bar_id");
```

Comment:
In `master`, the `ADD FOREIGN KEY` occurs before adding the destination column so it fails.

> <img width="1870" height="452" alt="image" src="https://github.com/user-attachments/assets/8e000492-1d6b-413e-b24d-3920da41ff94" />
>
> https://github.com/sqldef/sqldef/actions/runs/16308029916/job/46058064328